src/localCurrency/consts.ts

```diff
@@ -10,6 +10,7 @@ export enum LocalCurrencyCode {
   EUR = 'EUR',
   GBP = 'GBP',
   GHS = 'GHS',
+  GMD = 'GMD',
```

Comment:
These new currencies are not part of the fiatconnect spec, so we probably should not add them in https://github.com/fiatconnect/specification/blob/main/fiatconnect-api.md#921-fiattypeenum